.github/workflows/php.yml

```diff
@@ -0,0 +1,26 @@
+name: PHP CI
+
+on: push
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['7.1', '8.0']
```

Comment:
We should add: `8.1` and `8.2`

See: https://www.php.net/supported-versions.php

Won't add that much time to the builds